plugins/discourse-calendar/assets/javascripts/discourse/initializers/add-event-ui-builder.js

```diff
@@ -8,10 +8,14 @@ function initializeEventBuilder(api) {
 
   api.addComposerToolbarPopupMenuOption({
     action: (toolbarEvent) => {
+      const userTz = currentUser?.user_option?.timezone;
+      const timezone = userTz || moment.tz.guess() || "UTC";
```

Comment:
We might do it at other places but I don't `moment.tz.guess()` can return nothing

```suggestion
      const timezone = userTz || moment.tz.guess();
```